pubspec.yaml

```diff
@@ -81,13 +81,20 @@ dependencies:
   flutter_widget_from_html: 0.6.2
 
   # flutter_chips_input
+  # ISSUE: https://github.com/linagora/tmail-flutter/issues/131
+  # UPSTREAM ISSUE: TODO
+  # UPSTREAM PR: TODO
+  # EXPLANATION: We need a fork to be able to delete contacts from the composer
   flutter_chips_input:
     git:
       url: git://github.com/dab246/flutter_chips_input.git
       ref: master
 
   # html_editor_enhanced
-  html_editor_enhanced:
+  # ISSUE: https://github.com/linagora/tmail-flutter/issues/46
+  # UPSTREAM ISSUE: TODO
+  # UPSTREAM PR: TODO
+  # EXPLANATION: We customized to hide the toolbar of rich text editor to limit the functions out of our requirements
     git:
       url: git://github.com/dab246/html-editor-enhanced.git
```

Comment:
For this library we are not in final decision, so can we keep that state when we come back to refine composer next sprint? WDYT @chibenwa